plugins/net_plugin/net_plugin.cpp

```diff
@@ -265,7 +265,7 @@ namespace eosio {
    };
 
    const fc::string logger_name("net_plugin_impl");
-   fc::logger logger(logger_name);
+   fc::logger logger;
```

Comment:
So if a logger named "net_plugin_impl" doesn't appear in the config file, this nameless logger is the one that's used, and since it has no appenders, it acts as a silent sink for log messages?

Reply:
Yes, that's correct. Not saying it's the best approach.